src/game/GameScenario.ts

```diff
@@ -84,19 +85,18 @@ export class BasicGameScenario implements GameScenario {
      * @param action The player's choosen action for how to move forward in the game
      */
     getUpdatedState(
-        prevState: GameState,
```

Comment:
Outdated docs here too, Remove the `@param scenario` line

Reply:
This documentation var already out of date before these changes. This should be fixed in another PR